src/BbsBlsSignatureProof2020.ts

```diff
@@ -29,16 +29,17 @@ import { Bls12381G2KeyPair } from "@mattrglobal/bls12381-key-pair";
 export class BbsBlsSignatureProof2020 extends suites.LinkedDataProof {
   constructor({ useNativeCanonize, key, LDKeyClass }: any = {}) {
     super({
-      type:
-        "https://w3c-ccg.github.io/ldp-bbs2020/context/v1#BbsBlsSignatureProof2020"
+      type: "sec:BbsBlsSignatureProof2020"
```

Comment:
Can you try and see if this works with the full IRI instead? e.g `https://w3id.org/security#BbsBlsSignatureProof2020`

Reply:
These full IRI's didn't end up working because of the way the string matching works. We'll address this during the #81 though.